.test-infra/jenkins/job_LoadTests_Java.groovy

```diff
@@ -22,203 +22,225 @@ import LoadTestsBuilder as loadTestsBuilder
+                            tempLocation        : 'gs://temp-storage-for-perf-tests/loadtests',
+                            publishToBigQuery   : true,
+                            bigQueryDataset     : 'load_test',
+                            bigQueryTable       : "java_dataflow_${jobType}_GBK_6",
+                            sourceOptions       : """
+                                        {
+                                          "numRecords": 20000000,
+                                          "keySizeBytes": 10,
+                                          "valueSizeBytes": 90,
+                                          "numHotKeys": 200,
+                                          "hotKeyFraction": 1
+                                        }
+                                   """.trim().replaceAll("\\s", ""),
+                            fanout              : 1,
+                            iterations          : 4,
+                            maxNumWorkers       : 5,
+                            numWorkers          : 5,
+                            autoscalingAlgorithm: "NONE",
+                            streaming           : isStreaming
+                    ]
+            ],
+            [
+                    title        : 'Load test: reiterate 4 times 2MB values',
+                    itClass      : 'org.apache.beam.sdk.loadtests.GroupByKeyLoadTest',
+                    runner       : CommonTestProperties.Runner.DATAFLOW,
+                    jobProperties: [
+                            project             : 'apache-beam-testing',
+                            appName             : "load_tests_Java_Dataflow_${jobType}_GBK_7",
+                            tempLocation        : 'gs://temp-storage-for-perf-tests/loadtests',
+                            publishToBigQuery   : true,
+                            bigQueryDataset     : 'load_test',
+                            bigQueryTable       : "java_dataflow_${jobType}_GBK_7",
+                            sourceOptions       : """
+                                        {
+                                          "numRecords": 20000000,
+                                          "keySizeBytes": 10,
+                                          "valueSizeBytes": 90,
+                                          "numHotKeys": 10,
+                                          "hotKeyFraction": 1
+                                        }
+                                   """.trim().replaceAll("\\s", ""),
+                            fanout              : 1,
+                            iterations          : 4,
+                            maxNumWorkers       : 5,
+                            numWorkers          : 5,
+                            autoscalingAlgorithm: "NONE",
+                            streaming           : isStreaming
+                    ]
+            ]
         ]
-]
+    }
+
 
 def loadTestJob = { scope, triggeringContext ->
```

Comment:
Can we make `loadTestJob` more specific like `batchLoadTestJob` given we have `streamingLoadTestJob`?